tests/sympc/module/module_test.py

```diff
@@ -34,15 +34,15 @@ class ConvNet(sy.Module):
     def __init__(self, torch_ref, kernel_size=5):
```

Comment:
Q: Could you validate that stride was added to the module?

Reply:
Done.